### PR TITLE
Create it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

**Comment by daPhipz** [comment]
Cool!
You may want to add some metadata in the header. Look at the German translation as comparison :)

**Comment by Schneegans** [comment]
Thank you for the contribution! Do you want to finish the missing strings in this PR? Or shall we merge this first and you (or someone else) finish them later?

**Comment by albanobattistella** [comment]
Hi Simon,

If os possible to merge it now...because I want to test it and translate the other string with program are working..on the pc.

Thanks.

Albano
________________________________
Da: Simon Schneegans <notifications@github.com>
Inviato: giovedì 19 novembre 2020 10:57
A: Schneegans/Fly-Pie <Fly-Pie@noreply.github.com>
Cc: albanobattistella <albano_battistella@hotmail.com>; Author <author@noreply.github.com>
Oggetto: Re: [Schneegans/Fly-Pie] Create it.po (#43)


Thank you for the contribution! Do you want to finish the missing strings in this PR? Or shall we merge this first and you (or someone else) finish them later?

**Comment by Schneegans** [comment]
Hi Albano! While I _could_ merge it, I would rather prefer if you had tested this before on your side. Why can't you checkout your branch on your PC...? If you cloned https://github.com/Schneegans/Fly-Pie to `~/.local/share/gnome-shell/extensions` then you could add your fork as a second remote. Something along these lines (out of my head, not tested):

```bash
cd ~/.local/share/gnome-shell/extensions/flypie@schneegans.github.com
git remote add my-fork https://github.com/albanobattistella/Fly-Pie.git
git checkout -b my-develop my-fork/develop

# ... edit & test things ...

git push my-fork develop
```

Or you simply remove / rename the exiting `flypie@schneegans.github.com` and clone your fork to that location!

**Review by daPhipz — APPROVED** [review]

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,1724 @@
+# ITALIAN TRANSLATION FOR FLY-PIE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
```
Comment (daPhipz) [inline po/it.po:3]: You might want to add this additional info ;)

```suggestion
# Copyright (C) 2020 Simon Schneegans
# This file is distributed under the same license as the Fly-Pie package.
```

**Review by daPhipz — CHANGES_REQUESTED** [review]

File: po/it.po (hunk 1 above)
Comment (albanobattistella) [reply po/it.po]: > You might want to add this additional info ;)

Done!

**Comment by Schneegans** [comment]
Thank you so much! Is it OK, if I contact you in case that the translation needs an update?

**Comment by albanobattistella** [comment]
> Thank you so much! Is it OK, if I contact you in case that the translation needs an update?

Simon, will be a pleasure for me! ;-)

**Comment by daPhipz** [comment]
@Schneegans feel free to contact me in case you need a second pair of eyes for the German translation, too ;)
I will definitely stick around, so maybe I see changes myself. But ping me in case I miss something!

**Pull request merged**